From 50773b246cf39f59e93b553188419e3d9ccc1d7e Mon Sep 17 00:00:00 2001
From: rikardo <rikard.olander@hostmobility.com>
Date: Thu, 17 Jan 2019 13:52:59 +0100
Subject: [PATCH] Fix hang on SBC1 and SBC2 where clock set to 408Mhz

This patch is a similare fix that was done on 9ab298e for colibri and apalis 30.
fix also  sdmmc1 and sdmmc3 too to prevent error -22 on tegra_dvfs when it try to set 408MHz.
---
 arch/arm/mach-tegra/tegra30/clock.c | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/arch/arm/mach-tegra/tegra30/clock.c b/arch/arm/mach-tegra/tegra30/clock.c
index b366028..3e3b475 100644
--- a/arch/arm/mach-tegra/tegra30/clock.c
+++ b/arch/arm/mach-tegra/tegra30/clock.c
@@ -797,21 +797,6 @@ int tegra_plle_enable(void)
 }
 
 struct periph_clk_init periph_clk_init_table[] = {
-#if !defined(CONFIG_TARGET_APALIS_T30) && !defined(CONFIG_TARGET_COLIBRI_T30)
-	{ PERIPH_ID_SBC1, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SBC2, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SBC3, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SBC4, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SBC5, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SBC6, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_HOST1X, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_DISP1, CLOCK_ID_CGENERAL },
-#endif /* !CONFIG_TARGET_APALIS_T30 && !CONFIG_TARGET_COLIBRI_T30 */
-	{ PERIPH_ID_NDFLASH, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SDMMC1, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SDMMC2, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SDMMC3, CLOCK_ID_PERIPH },
-	{ PERIPH_ID_SDMMC4, CLOCK_ID_PERIPH },
 	{ PERIPH_ID_PWM, CLOCK_ID_SFROM32KHZ },
 	{ PERIPH_ID_DVC_I2C, CLOCK_ID_PERIPH },
 	{ PERIPH_ID_I2C1, CLOCK_ID_PERIPH },
-- 
2.7.4

